src/player.js

```diff
@@ -286,6 +299,33 @@ class Player extends FakeEventTarget {
 
   //  <editor-fold desc="Playback Interface">
   /**
+   * The player readiness
+   * @public
+   * @returns {Promise<*>} - The ready promise
+   */
+  ready(): Promise<*> {
+    return this._readyPromise ? this._readyPromise : Promise.resolve();
```

Comment:
in destroy create new promise which is not resolved.
`this._readyPromise = new Promise();`